client/packages/common/src/ui/layout/tables/material-react-table/usePaginatedMaterialTable.tsx

```diff
@@ -119,6 +127,53 @@ export const usePaginatedMaterialTable = <T extends MRT_RowData>({
     manualFiltering: true,
     manualPagination: true,
     manualSorting: true,
+    enableBottomToolbar: true, // required for pagination
+    enablePagination: true,
+    paginationDisplayMode: 'pages',
+    muiBottomToolbarProps: {
+      sx: {
+        '& .MuiInputLabel-root': {
+          fontSize: '0.9em',
+        },
```

Comment:
The `muiPaginationProps` targets the pagination component only, not the rows selector, so have to target this via the outer Toolbar component